step-definitions/standard-ui/button.js

```diff
@@ -25,6 +25,21 @@ When('I click on button {string}',
   }
 );
 
+/**
+ * Click on button with a certain content.
+ */
+When('I click on button with id {string}',
+  async (content) => {
+    // xpath will be used as the locating strategy so all the selectors you pass should be xpath selectors
+    client.useCss();
```

Comment:
shouldn't be necessary to set explicitly, also comment doesn't match the functionality ;)

Reply:
but when before is xpath used, then the test will fail... had that one